packages/common/src/utils/filters.ts

```diff
@@ -421,6 +421,125 @@ export const getFilterRulesByFieldType = (
         },
     );
 
+const flattenFilterGroup = (filterGroup: FilterGroup): FilterGroup => {
```

Comment:
A quick comment explaining what the method does up top would be a good extra touch 💅